Posit Infrastructure/Posit Environment Quick Start Guide.md

```diff
@@ -0,0 +1,249 @@
+
+![NIGHT session executing](https://github.com/user-attachments/assets/b50c0dab-3950-4c83-9b8a-ea76f7faef43)
+
+On completion, and once the session has closed successfully, the session will be reported as having `Finished`:
+
+![NIGHT session finished](https://github.com/user-attachments/assets/0ef26b7f-cf91-4887-b43c-35691aff1cc0)
+
+---
+## 6. Configure Git and GitHub
+
+The following is only relevant if you/your team use Git in your work (i.e. collaborative work using repositories hosted on Github):
+
+### Configure Git
+
+Add your name and email to Git by running the following commands in the terminal:
+
+```
+git config --global user.email <your email address>
+git config --global user.name <Your full name>
+```
+
+### Create an SSH key
+In the **Git/SVN** tab, hit *Create RSA Key* (A). 
+In the window that appears, hit the *Create* button (B). 
+Close this window.
+
+Click, *View public key* (C), and copy the displayed public key (D).
+![](https://github.com/SurgicalInformatics/healthyr_book/blob/8da2d63f5ac55e1734b2ca9afdc800d17c5401cb/images/chapter14/1.png)
+
+### Add the SSH key to GitHub
+On GitHub, log in, open the account settings page and go to the [SSH keys tab](https://github.com/settings/keys) (A). 
+Click *Add SSH key* and paste the public key you copied from RStudio (B).
+
+![](https://github.com/SurgicalInformatics/healthyr_book/blob/8da2d63f5ac55e1734b2ca9afdc800d17c5401cb/images/chapter14/2.png)
+
+---
+## 7. Getting Support
+
+The support process for the Posit Environment is as follows:
+
+1. All requests for support and to report issues with the Posit Environment must be made via the form at https://forms.office.com/e/pqmmNMkhT6
+
+These requests for support will first reach the Data Science Team, who will provide first-line support where possible.  If the Data Science Team cannot resolve the issue, it is passed to second-line support.
+
+2. Second-line support for the Posit Environment will be provided by Jumping Rivers.  An Engineer or Data Scientist at Jumping Rivers may contact you directly about the issue you are experiencing.
+
+3. NSS DaS will only become involved in exceptional circumstances, such as a network outage, or if a significant change is required in the Microsoft Azure cloud computing environment.  The Data Science Team will coordinate the communication between Jumping Rivers and NSS DaS in these instances.
+
+**Important Notes**
+
+- You must complete the form linked above to receive support relating to the Posit Environment.
+- Do not contact Jumping Rivers, unless they request further information from you directly.
+- Do not raise a request or incident in Service Now, unless directed to do so by the Data Science Team.
+
+---
+
+## 8. FAQs
+### What if I encounter a problem?
+- Contact the Data Science Team for support using the form at [https://forms.office.com/e/pqmmNMkhT6]
```

Comment:
Angle brackets are proper markdown for a bare link like this.
```suggestion
- Contact the Data Science Team for support using the form at <https://forms.office.com/e/pqmmNMkhT6>
```

The square brackets were rendering as literal square brackets.